src/test/kotlin/javabot/BaseTest.kt

```diff
@@ -77,15 +77,16 @@ open class BaseTest {
 
     @BeforeTest fun setup() {
         messages.clear()
-        var admin = adminDao.getAdminByEmailAddress(BOT_EMAIL)
-        if (admin == null) {
-            admin = Admin(testUser.nick, BOT_EMAIL, testUser.hostmask, true)
-        } else {
-            admin.ircName = testUser.nick
-            admin.emailAddress = BOT_EMAIL
-            admin.hostName = testUser.hostmask
-            admin.botOwner = true
+        var admin = try {
+            adminDao.getAdminByEmailAddress(BOT_EMAIL)
+        } catch(adminNotFound: RuntimeException) {
+            Admin(testUser.nick, BOT_EMAIL, testUser.hostmask, true)
         }
+        admin.ircName = testUser.nick
```

Comment:
These next assignment lines are redundant now.  Also, nice use of the try-as-an-expression feature of kotlin.  :)